Fixed the pip install command's package name

stevenmz opened the pull request:
Found the correct package name at https://pypi.org/project/py-pgp/

stevenmz committed:
Fixed the pip install command's package name

Found the correct package name at https://pypi.org/project/py-pgp/

fzhengwx commented:
@stevenmz Can you install this package? If yes, what's your python version? 

Does this library provide real full functionality of GnuPG without any external system dependency? For instance, all other similar python libraries are all wrapper around the GnuPG executable.

stevenmz commented:
Hi @fzhengwx ! Yes, I am able to install the package using the package name I modified it to in the pull request. I am on Anaconda’s Python 3.6.7. I looked at this package first because it did not require a pgp install, but I found it’s API a bit harder to use than other packages (for my basic need of reading encrypted files) so I switched.
